<launch>
    <node name="rome_ekf_wrapper" exec="rome_ekf_wrapper" pkg="mav_drone_localization" args="--ros-args --log-level warn">
        <param name="use_sim_time" value="true" />
    </node>
    <node pkg="robot_localization" exec="ekf_node" name="ekf_filter_node_odom">
        <param from="$(find-pkg-share mav_drone_localization)/config/ekf_simplest_try.yaml"/>
        <param name="use_sim_time" value="true" />
        <remap from="/odometry/filtered" to="/odometry/filtered/local"/>
    </node>
    <node pkg="robot_localization" exec="ekf_node" name="ekf_filter_node_map">
        <param from="$(find-pkg-share mav_drone_localization)/config/ekf_simplest_try.yaml"/>
        <param name="use_sim_time" value="true" />
        <remap from="/odometry/filtered" to="/odometry/filtered/global"/>
    </node>
    <!-- <node pkg="robot_localization" exec="navsat_transform_node" name="navsat_transform" respawn="true">
        <param from="$(find-pkg-share mav_drone_localization)/config/ekf_simplest_try.yaml"/>
        <remap from="/imu/data" to="/mavros/imu/data" />
        <remap from="/gps/fix" to="/mavros/global_position/raw/fix" />
        <remap from="/odometry/filtered" to="/odometry/filtered/global" /> 
        <remap from="/datum" to="/navsat_transform/set_datum" /> 
    </node> -->
</launch>